gke-usage-metering/src/gkeUsageMetering.ts

```diff
@@ -141,7 +141,10 @@ namespace gkeUsageMetering {
         .setText('The resource usage dataset ID is not specified')
         .throwException();
     }
-    if (fullResourceUsageDataset.indexOf('.') < 0) {
+    // GCP project name can contain letters, numbers, single quotes, 
```

Comment:
Can you mention here that `fullResourceUsageDataset` should be in the format of `${PROJECT_ID}.${DATASET_ID}`. 

Also we should say "project ID" here instead of "project name", because they are actually different identifiers.

Reply:
Done